beeline/middleware/flask/__init__.py

```diff
@@ -26,17 +26,17 @@ def __init__(self, app):
         self.app = app
 
     def __call__(self, environ, start_response):
-        trace_name = "flask_http_%s" % environ['REQUEST_METHOD'].lower()
+        trace_name = "flask_http_%s" % environ.get('REQUEST_METHOD', None).lower()
```

Comment:
this might be a problem. can't lower None

Reply:
oops!